cloud/services/compute/networks/reconcile.go

```diff
@@ -102,6 +108,11 @@ func (s *Service) createOrGetNetwork(ctx context.Context) (*compute.Network, err
 			return nil, err
 		}
 
+		if s.scope.IsSharedVpc() {
+			log.Error(err, "Shared VPC is enabled, but could not find find existing network", "name", s.scope.NetworkName())
```

Comment:
Nit: "find" occurs twice in this error message.

Reply:
I only see this occur once. If you are referring to a similar error message, one is for the routers and one is for the network.